Team417_CENTERSTAGE/src/main/java/org/firstinspires/ftc/team417_CENTERSTAGE/roadrunner/MecanumDrive.java

```diff
@@ -570,9 +558,12 @@ public void runParallel(Action action) {
         actionList.add(action);
     }
 
+    // Because apparently Action.run can't have a null TelemetryPacket
+    TelemetryPacket placeholderTelemetryPacket = new TelemetryPacket();
+
```

Comment:
Every call to `doActionsWork()` adds stuff to the telemetry packet so over time it will get bigger and bigger until it consumes all of memory. If you really want a throwaway placeholder like this, create a new one for every call (the garbage collector will free the memory from the old one). Better yet though would be to always have the caller create the telemetry packet and the caller should send it to FTC Dashboard by calling `FtcDashboard.getInstance().sendTelemetryPacket(packet)` so you can actually see the results.